.builders/images/linux-x86_64/Dockerfile

```diff
@@ -21,6 +21,7 @@ ENV LD_LIBRARY_PATH="/opt/python/${PYTHON2_VERSION}/lib:${LD_LIBRARY_PATH}"
 # Set up virtual environment for Python 2
 RUN /opt/python/${PYTHON2_VERSION}/bin/python -m pip install --no-warn-script-location virtualenv \
  && /opt/python/${PYTHON2_VERSION}/bin/python -m virtualenv /py2
+ENV DD_PY2_BUILDENV_PATH=/py2/bin/python
```

Comment:
Since we've already set the default in the script these seem superfluous. Can we remove these or require them? I would also like Windows to be consistent.

Reply:
I've removed them from the Dockerfile so that only the mac OS build uses these variables currently? With this approach is consistency with Windows still a concern? Would you rather make them required?